src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -773,6 +781,12 @@ private static void SubstituteConstraintTypes(ImmutableArray<TypeWithAnnotations
                     return false;
                 }
 
+                if (considerParamsAndDefaultValues && (param1.IsParams != param2.IsParams ||
```

Comment:
>param1.IsParams != param2.IsParams [](http://example.com/codeflow?start=55&length=34)

I think `IsParams` value matters only on the last parameter, therefore we probably do not want to fail due to the mismatch on some other parameter.